test/release/examples/primers/modules.chpl

```diff
@@ -297,6 +404,10 @@ module MainModule {
       // Should output 3 - 2, or '1'
     }
 
+    /* Due to the definition of ``import`` statements, only the module name or
+       specific symbols within the module are brought in.
+    */
```

Comment:
I found this sentence a bit... adrift?

Maybe it can be above along with other `import` descriptions as this last part steered away from specifics of `use` and `import`.

I also find the wording a bit confusing. Do you mean the below code is not allowed:

```chapel
import A, B.myFunc, C.myVar;
```

or do you mean that
```chapel
import A.foo;
```
doesn't actually bring in the name `A`?

I think you mean the second limitation here. If so it can probably be added as a single sentence to the part where you explain `import A.foo;` that this statement by itself doesn't bring the name `A` to the scope

Reply:
You're correct, I do mean the latter (though the former is certainly true).  I mostly intended this to remind users of something that should have been mentioned earlier, since it is a contrast to use statements.

Reply:
I think I've improved it, mind taking a look at the changes now?  I've also pushed an update to the web docs

Reply:
Yeah this one makes sense too.